Open-XML-SDK.sln

```diff
@@ -57,6 +57,16 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DocumentFormat.OpenXml.Fram
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DocumentFormat.OpenXml.Linq.Tests", "test\DocumentFormat.OpenXml.Linq.Tests\DocumentFormat.OpenXml.Linq.Tests.csproj", "{3B82C340-3B10-42BF-9A2C-2E2327A9CFD4}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AddAnimatedModel3D", "samples\AnimatedModel3DExample\AddAnimatedModel3D.csproj", "{12BFA5FC-F09A-480D-BBAC-3992B23B277D}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "doctaskssample", "samples\DocumentTaskExample\doctaskssample.csproj", "{6B7EBFFD-8FD9-47F1-9F2F-68943FA1DF42}"
```

Comment:
`doctaskssample` has abbreviations. Can you rename it to `DocumentTasksSample` using PascalCase like the other samples

Reply:
done